examples/web/index.html

```diff
@@ -313,6 +313,142 @@ <h4 class="typography-heading-xsmall-text">Heading XSmall Text</h4>
           </p>
         </div>
       </section>
+
+      <section>
+        <div class="Container">
+          <h2>CheckboxField</h2>
+          <div class="example-CheckboxField-grid">
+            <label for="checkboxfield1" class="CheckboxField">
+              <input type="checkbox" id="checkboxfield1" class="CheckboxField__input" />
+              <div class="CheckboxField__text">
```

Comment:
`<div>` elements shouldn't be used inside `<label>` tag. Can we avoid that? For example by using `<span>`.

Reply:
Ok, changed to `<span>`.